content/09.overfitting.md

```diff
@@ -1 +1,21 @@
 ## Tip 7: Address deep neural networks' increased tendency to overfit the dataset {#overfitting}
+
+Overfitting is one of the most significant dangers faced by a deep learning practitioner.
+Put simply, overfitting occurs when a model fits patterns in the training data too closely, includes noise or non-scientifically relevant perturbations, or in the most extreme case, simply memorizes patterns in the training set.
+This subtle distinction is made clearer by seeing what happens when a model is tested on data to which it was not exposed during training: just as a student who memorizes exam materials struggles to correctly answer questions for which they have not studied, a machine learning model that has overfit to its training data will perform poorly on unseen test data.
+Deep learning models are particularly susceptible to overfitting due to their relatively large number of parameters and associated representational capacity.
+To continue the student analogy, a smarter student has greater potential for memorization than average one and thus may be more inclined to memorize.
```

Comment:
Like @chevrm, I think this analogy could be misleading.  If you and @rasbt like it, I'm fine leaving it in this draft, but I may want to revisit it later.

Reply:
Please do! Let's get this merged and either you or @chevrm can PR changes/clarifications as y'all see fit.